third-party/libhdfs3/Makefile

```diff
@@ -0,0 +1,35 @@
+ifndef CHPL_MAKE_HOME
+export CHPL_MAKE_HOME=$(shell pwd)/../..
+endif
+
+CHPL_MAKE_HOST_TARGET = --host
+include $(CHPL_MAKE_HOME)/make/Makefile.base
+include Makefile.share
+
+LIBHDFS3_FILE := libhdfs3/bootstrap
+
+default: $(LIBHDFS3_FILE)
+
+all: $(LIBHDFS3_FILE)
+
+clean: FORCE
+	rm -rf libhdfs3
+
+cleanall: FORCE
+	rm -rf build
+
+clobber: FORCE
+	rm -rf build install llvm
+
+$(LIBHDFS3_FILE):
+	if [ ! -d llvm ]; then ./fetch-libhdfs3.sh; fi
```

Comment:
yep, going back through and cleaning up the build scripts - had an unclean synch with origin/master and the backups had some errors - apologies for not QCing this thoroughly before the push.